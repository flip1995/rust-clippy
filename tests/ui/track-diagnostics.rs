// compile-flags: -Z track-diagnostics
// error-pattern: created at

struct A;
struct B;
const S: A = B;

fn main() {}
