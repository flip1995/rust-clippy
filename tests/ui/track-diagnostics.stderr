error[E0308]: mismatched types
  --> $DIR/track-diagnostics.rs:$LINE::$COL
   |
LL | const S: A = B;
   |              ^ expected struct `A`, found struct `B`
-Ztrack-diagnostics: created at compiler/rustc_infer/src/infer/error_reporting/mod.rs:$LINE::$COL

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
